packages/analytics/src/send-pingback.ts

```diff
@@ -1,8 +1,10 @@
 import { getGiphySDKRequestHeaders, Logger } from '@giphy/js-util'
 import { PingbackEvent } from './types'
+import gl from './global'
 
 // TODO remove api key
-const pingBackUrl = 'https://pingback.giphy.com/v2/pingback?apikey=l0HlIwPWyBBUDAUgM'
+const environment = gl?.GIPHY_PINGBACK_URL ?? 'https://pingback.giphy.com'
```

Comment:
@dleitee `??` would mean 0, false, or '' are valid. Maybe '' is valid? Otherwise just use `||`